### DISCUSSION TITLE
woowacourse/woowacourse-level3-refresh-token/

### DISCUSSION THREAD

**Discussion by giscus** [chlwlstlf/chlwlstlf.github.io#4] [Announcements]
# woowacourse/woowacourse-level3-refresh-token/

Refresh Token으로 Access Token 갱신하기

https://chlwlstlf.github.io/woowacourse/woowacourse-level3-refresh-token/

**Comment by chysis** [comment]
의사 결정 과정과 프로젝트를 진행하면서 겪었던 문제 상황이 자세하게 언급되어 있어 좋았어요 👍🏻

**Reply by chlwlstlf** [reply]
글 읽어주셔서 감사합니다😁 이 부분에 대해 모의 면접을 했더니 쓸 내용이 더 많아졌네요ㅎㅎㅎㅎㅎㅎ
